button update frenzy & former officer updates

ryankbr opened the pull request:
closes #232 

all buttons open to new tabs,
acc button at the top refreshes the page on click

closes #229 

all former officer descriptions updated & dakshika's status updated

ryankbr committed:
button update frenzy

ryankbr committed:
updated all former officer descriptions

Sofia-Ortega commented on src/data/officers.json:
```diff
@@ -90,159 +90,160 @@
       "position": "Events and Marketing Officer",
       "term": "2022",
-      "description": "Jay is a Psychology Computer Science double major. In her free time she trains service dogs and rock climbs!"
+      "description": "Jay is a Psychology / Computer Science double major. In her free time she trains service dogs and rock climbs! She is now a Campus Ambassador for Bartleby!"
```
idk if this is still true (the fact that she is a Campus Ambassador for Bartleby

ryankbr replied:
yeah:D i noticed yesterday that jay posted about the bartlelby thing plus that was her current role on linked in so i added it this way:o

Sofia-Ortega commented on src/data/officers.json:
```diff
@@ -90,159 +90,160 @@
-        "description": "Anthony is a senior Computer Science major who loves producing and playing music. When he’s not coding, he enjoys hiking, playing soccer, and watching Formula 1."
+      "name": "Anthony Teo",
+      "image": "/static/images/former-officers/anthony_teo.jpg",
+      "major": "Computer Science '22 (TAMU), M.S. Human-Computer Interaction (CMU) '23",
```
When you add MS here, it looks too blocky and overwhelming on the website. Like, its difficult for the human eye to process / scan

ryankbr replied:
that makes sense-- it does look a little weird so ill see if i can think of a better way to represent this:o maybe just mentioning it in the description would be good

Sofia-Ortega commented on src/data/officers.json:
```diff
@@ -90,159 +90,160 @@
       "position": "Projects Chair",
       "term": "2020 - 2021",
-      "description": "Gannon is a Senior Computer Science student, whose interests include creating useful products and project management as a whole! When he’s not programming, he likes to play video games, play basketball, and watch film!"
+      "description": "When Gannon's not programming, he likes to play video games, play basketball, and watch film! He is now a Software Engineer II at Amazon working on iOS development for eero!"
```
I dont think Gannon**'s** is correct grammer. Correct me if im wrong tho

ryankbr replied:
ahh yeah thats supposed to be a contraction for Gannon is but i can just expand it so that it's clear

Sofia-Ortega commented on src/data/officers.json:
```diff
@@ -90,159 +90,160 @@
       "name": "Feras Khemakhem",
       "image": "/static/images/former-officers/feras_khemakhem.jpg",
-      "major": "Computer Science",
+      "major": "B.S. Computer Science '20, M.S. Computer Science '22",
```
Same thing about header looks too clunky now. Its just difficult to scan. I'm not saying dont add it, but remove it for now, and see if you can find another way so that adding masters can be more scannable

ryankbr replied:
same as above o-o will probably move this to description in that case

Sofia-Ortega commented on src/views/index.html:
```diff
@@ -22,14 +22,15 @@
       <div class="container-fluid">
         <div class="navbar-header">
-          <a class="navbar-brand" href="#">
+          <a class="navbar-brand" href="#" onClick="window.location.reload()">
```
why did you do this?

ryankbr replied:
this reloads the webpage when you click on the top logo which mimics the apple website:D

Sofia-Ortega commented on src/views/index.html:
```diff
@@ -22,14 +22,15 @@
           <li><a href="#">Home</a></li>
           <li><a href="#how-to-join">Join</a></li>
           <li><a href="#projects">Projects</a></li>
+          <li><a href="#officer-team">Officers</a></li>
```
Hmmm, I am unsure if we want to add Officers to the Navbar.

The Navbar is precious space, and we want it to highlight what is absolutely important to the average visitor. Why would they need to know about the officers?

Let me know your thoughts

ryankbr replied:
> Also, what does this mean, "acc button at the bottom brings to top of page,"

the acc logo at the bottom i made into a button that will like scroll to the top of the page :) like a "to top button"

ryankbr replied:
woops this last one was a reply to the last comment LOL

ryankbr replied:
but i dropped the question in officers o-o can remove it :o everything else is about ready to go

Sofia-Ortega commented on src/views/index.html:
```diff
@@ -147,7 +148,7 @@ <h1>HOW TO JOIN</h1>
             </p>
             <div class="center-small">
-              <a href="https://discord.gg/tfu5SngfK9">
+              <a href="https://discord.gg/tfu5SngfK9" target="_blank">
```
i like this

Sofia-Ortega reviewed (changes requested):
Overall, I like what you did as a whole with updating the officer descriptions. It must have taken a lot of research.

Also, I like how you a lot of the links now open to a new page.

Sofia-Ortega commented:
Also, what does this mean, "acc button at the bottom brings to top of page,"

ryankbr committed:
diff review revisions for former officer descriptions

ryankbr committed:
commented out officer navbar link

Sofia-Ortega commented on src/views/index.html:
```diff
@@ -369,9 +373,11 @@ <h1>GALLERY</h1>
 
     <footer>
       <div class="container">
+          <a href='#'>
         <img class="acc-logo" src="../static/images/logo-acc.png" />
+          </a>
```
ok, I am now confused why you have it scroll up to the top.

Will the user expect this? Is this a common behaviour?

ryankbr replied:
oh yeah:o since the header doesn't stick or like pop back up when you scroll down, its for like fast navigation back to the top especially for like mobile devices where you have to physically scroll up to the top o-o

this will pretty much let them click the button and instead of refreshing, they'd be pulled back up to the top

Sofia-Ortega replied:
Like, I get what it DOES. But I was wondering if it was an industry standard? 

My internship in the summer kinda drilled into me "what does the user expect," so now Im traumatized 😅

Sofia-Ortega commented on src/views/index.html:
```diff
@@ -22,14 +22,15 @@
           <li><a href="#">Home</a></li>
           <li><a href="#how-to-join">Join</a></li>
           <li><a href="#projects">Projects</a></li>
+          <!-- <li><a href="#officer-team">Officers</a></li> -->
```
Can u just delete this instead? It keeps the code base clean. You can always add it back later.

carolgeng commented on src/data/officers.json:
```diff
@@ -90,159 +90,160 @@
       "position": "Project Management Chair",
       "term": "2017 - 2020",
-      "description": "Feras specializes in Computer Graphics and animation, and hopes to continue this work into the field. Personal hobbies include traveling to eat good food, and gaming."
+      "description": "Feras completed his Masters in Computer Science in 2022 at A&M working on Computer Graphics and Animation the field. Personal hobbies include traveling to eat good food, and gaming. He is now a Software Engineer at Microsoft!"
```
I think we can remove "the field" and also change 'Personal hobbies' to  'His personal hobbies'

ryankbr committed:
removed comment & "to-top" functionality

went ahead and removed the commented out navbar code to keep the codebase clean and also removed the to top functionality, i'll instead maybe try and make a sleek transparent "top" floater button that stays at the bottom for easy navigation or have a collapsible nav bar:)

Sofia-Ortega reviewed (approved):
Amazing 😮 😎 🥳

ezrj reviewed (approved):
what sofia says goes
